packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -146,6 +164,7 @@ export function createOdspNetworkError(
         }
     }
     error.addTelemetryProperties(props);
+    assert(error.facetCodes === facetInfo.facetCodes, "facet codes are wrong");
```

Comment:
This will be always true, because code above does not operate with property by name "facetCodes", only with property "facetCode" and it's not array (just single code)